### ISSUE TITLE
待翻：在加入创业公司之前需要去问创始人的7个问题（7 Questions to Ask Founders Before Joining Their Startup）

### ISSUE THREAD

**Issue by lietoumai** [lietoumai/Hunter#25]
硅谷的顶尖人才有很多职业选择，他们通常可以在大公司赚到更多的钱，但还是愿意加入创业公司，原因如下:

每天都很有意义；用小时来衡量进度，而不是用周。
在产品和客户体验上有很强的影响力
办公室政治少
与有能力、有积极性的团队合作

上面那些听起来都不错，但是加入创业公司需要考虑的更多。当你在选择创业公司加入的时候，我们还应该看看创业成功的可能性。这是我在这篇文章中将要讨论的内容。

You may have heard something along the lines of “Most startups fail, so just join a team you like and enjoy wherever it goes.” or “The equity is just a bonus in case something comes of it.” These assume you are incapable of assessing a startup’s potential, which is an incorrect assumption.

Startups are a game of skill, not of chance.

Startups are not lottery tickets
Venture capital firms yield a wide range of annual return rates. The top funds can return a rate of over 20% while the median is close to 10% and the bottom venture capital firms may produce a negative return. If startups were all just lottery tickets then there wouldn’t be such a consistently different rate of return among venture capital firms.
One may believe that venture capitalists have the inside track to differentiate startup potential and that mere mortals roaming Silicon Valley do not. While it’s true you will probably not be better than the top individual venture investors, it doesn’t mean that you’re helpless in assessing a startup’s potential.

The following are 7 questions that you should ask any startup you are looking to join. The range of answers will provide significant insight into the startup’s potential.
7 questions to ask when considering a startup’s potential
How much has the company raised and what is its runway?
Who are your investors and why did you choose them?
Who are the founders and what are their qualifications to run a startup?
What is the board structure?
Who are the early employees and what are their backgrounds?
What milestones has the company achieved?
Ask yourself: Do I believe in the startup’s idea?

What to look for when asking startups the 7 questions
How much has the company raised and what is its runway?
Raising money and succeeding are two very different things, but Techcrunch analysis shows though that the odds of raising a Series A goes up significantly when a startup raises $1.5M or more in seed.

From Techcrunch — Does it really matter how much your startup raises?
While it may be easy to take comfort that there is at least 7 figures in the startup’s bank account, something often overlooked is asking about the startup’s burn rate. If two startups have $2M in the bank and one has a burn rate of $50,000/mo and one has a burn rate of $300,000/mo, the former has a runway of 40 months while the latter has less than 7 months before running out of cash.
So follow up questions to ask are “How much is in the bank now? What is the burn rate? What do you project as your runway?” — These are reasonable questions to ask, so don’t be uncomfortable is asking them. If the company can’t trust prospective core team members then it sets a foundation for a rather weak relationship.
Alternatively, the startup may not consider you to be a core team member, which is also an important takeaway.
Who are your investors and why did you choose them?
In addition to how much money the startup raised is a question of where the money comes from. If it’s from a top venture capital firm then that means the startup has overcome at least some early startup obstacles. If the investment is mostly from a founder’s uncle, that’s still better than no funding but it’s less indicative of likely success. The startup should at the very least be able to explain to you why they chose their investors.
Who are the founders and what is their qualifications to run a startup?
This is the most important factor when venture capitalists consider a startup, so it should be high on your list. Obvious things to consider are overall experience, startup experience, education, and their temperament and motivation. Ultimately, you should decide whether the founders impress you. Is the CEO somebody you believe you can learn from every day?
Be careful not to fall into stereotype assumptions. The average successful CEO is not a 20something college dropout. The average age is 34–40, most likely attended Harvard, Stanford, or Berkeley, and works in a team of 2–3 founders who previously worked together where at least one is serial entrepreneur. These are definitely not checkboxes to try to fill, as successful founders come from all sorts of backgrounds. The point is not to stereotype and instead focus on whether the founders are people who impress you and whether you believe you can learn from.
What is the board structure?
If a startup’s capital comes from the founder’s uncle, that is ok… sometimes that just makes things easier to get started. But if that uncle is involved in the business, or even worse, on the Board of Directors, that should be a non-starter red flag.
Board structure is very important in any company — it determined who sets the business direction and top leadership. The default (neutral) case is that the founders make up the board. The downside is when an unqualified family member or unqualified angel investor is on the board. Consider it an upside when the founders brought in a great board member to their team.
Who are the early employees and what are their backgrounds?
Top performers want to work with other top performers, so pay special attention to who the early team members are. Some startups will take shortcuts and compromise on hiring just to get the work done because there is a sense of urgency for progress.
We certainly feel this pressure at CarDash. Our slow hiring is at times at odds with our desire for quick growth. However, hiring too quickly to meet short term goals is likely to compromise the long term culture of the company.
There is certainly a balance to find here. A good rule of thumb I follow is “if the next 10 people I hire for this position would be this person”, would I do it? If the answer is no, then I don’t believe it’s worth making the compromise otherwise it would be too easy to compromise time and time again.
What milestones has the company achieved?
The value in this question is less about the milestones themselves and more about revealing what the founders value. Strong responses are things like “We’ve reached $XXX in recurring monthly revenue” or “We’ve signed on company X and company Y as paid customers” or “We have XXX weekly active users, which we define as…” etc. All terms that are business focused.
I would be more wary of founders who respond with “We were profiled in Techcrunch” or “Our web site had XXX visitors last month”. These are vanity measurements and may forecast a vanity oriented method of management.
Being part of Y Combinator is something CarDash is incredibly proud of, but that’s not a business milestone in and of itself, it is only a way to accelerate business milestones. The same is true for fundraising.
Ask yourself: Do I believe in the startup’s idea?
This is initially a question for yourself. If the startup is in a space foreign to you then the founders should be able to sell you on the idea. If they can’t sell an early employee, they likely won’t be able to sell it to prospective customers either. If it’s a space you have an intuition for, then ask yourself if the idea resonates with you. If it doesn’t, give the founders a chance to sell you on the broader vision. If it still doesn’t make sense to you, you really owe it both to yourself and the company to pass. Early stage teams need to believe in what they are working on.
At CarDash, everybody who has joined us understands how broken the automotive service industry is and wants to modernize and improve a customer service experience that desperately needs to change.

In summary
Joining an early stage start should be a commitment of several years, often in the prime of your career. The decision must therefore be taken with the highest of care; it’s very different than taking a job at some company with a convenient commute and a nice benefits program. The work will be long, hard, and hopefully the most rewarding of your career. The 7 questions highlighted in this post shouldn’t take more than 15 minutes to cover. You owe it to yourself to go through the diligence before committing some of the best years of your life to a startup.

**Comment by jixi1991** [comment]
mark

**Comment by hunterhug** [comment]
good
